trace.go

```diff
@@ -0,0 +1,15 @@
+package runn
+
+type trace struct {
+	RunID string `json:"id"`
+}
+
+func NewTrace(o *operator) trace {
+	tls := o.trails()
+
+	return trace{
+		// TODO: Create IDs that include even query strings
+		// ex) [runbook ID]?step=2&step=0
+		RunID: tls[len(tls)-1].RunbookID,
```

Comment:
See https://github.com/k1LoW/runn/pull/646#issuecomment-1766699471

Reply:
Would it be something like this?
[use operator.runbookIDFull](https://github.com/k1LoW/runn/pull/645/commits/ea999d4282e7d70457d29f8689b4edde9815c48e)